syftlib/src/main/java/org/openmined/syft/domain/SyftConfiguration.kt

```diff
@@ -23,7 +23,7 @@ class SyftConfiguration internal constructor(
     val cacheTimeOut: Long,
     val maxConcurrentJobs: Int,
     private val socketClient: SocketClient,
-    private val httpClient: HttpClient,
+    val httpClient: HttpClient,
```

Comment:
Is there any reason to make this public? We should be using `getDownloader()` for accessing it

Reply:
Good catch!